doc/related_projects.rst

```diff
@@ -12,6 +12,12 @@ Interoperability and framework enhancements
 These tools adapt scikit-learn for use with other technologies or otherwise
 enhance the functionality of scikit-learn's estimators.
 
+- `Machine Learning <https://github.com/jeff1evesque/machine-learning>`_ an automated build
```

Comment:
The corresponding ubuntu vm, is containerized using vagrant, while puppet provisions it.  Assuming both virtualbox, and vagrant is installed, with proper github settings, one command builds the entire application:

```
$ cd /path/to/project
$ vagrant up
```